templates/etc/haproxy/logforward.cfg.j2

```diff
@@ -0,0 +1,24 @@
+{% if haproxy_version is version('2.3', '<=') %}
```

Comment:
Should this be `>=`?

https://www.haproxy.com/blog/log-forwarding-with-haproxy-and-syslog